packages/vite/src/node/server/index.ts

```diff
@@ -369,6 +370,10 @@ export interface ViteDevServer {
    * @internal
    */
   _configServerPort?: number | undefined
+  /**
+   * @internal
+   */
+  _moduleType?: string | undefined
```

Comment:
Should the type of `_moduleType` be directly restricted to `'module' | 'commonjs' | undefined`?